etc/webapi.xml

```diff
@@ -48,4 +48,22 @@
             <resource ref="anonymous"/>
         </resources>
     </route>
+    <route url="/V1/klaviyo/reclaim/log" method="GET">
```

Comment:
Hi @njparadis,
Could you explain why it could be needed - managing logs via REST API?
It looks to me as security issue because in logs might be cleaned via rest api.

Reply:
These methods only add or remove messages from our own custom log file added in this PR. By default the logger is disabled.

The idea behind these endpoints is to help debug issues with the extension without needing to wait for the customer to send over the log files. The clean method will be called as part of the periodic sync our app performs to ensure that the log file does not grow to be too large.